_docs/javascript-api.md

```diff
@@ -911,6 +911,32 @@ Interceptor.attach(Module.findExportByName("libc.so", "read"), {
 
     -   `depth`: call depth of relative to other invocations
 
+    For eaxmple:
+
+{% highlight js %}
+Interceptor.attach(Module.findExportByName(null, 'read'), {
+    onEnter: function (args) {
+        console.log('Context information: ')
+        console.log("Context  : " + JSON.stringify(this.context))
+        console.log("Return   : " + this.returnAddress)
+        console.log("ThreadId : " + this.threadId)
+        console.log("Depth    : " + this.depth)
+        console.log("Errornr  : " + this.err)
+        console.log("Arg 0    : " + args[0])
+        //Save argument 0, for processing in onLeave.
+        this.args = []
+        this.args[0] = args[0]
```

Comment:
I would simplify this to: `this.arg0 = args[0]`. Actually, since this example is about the `read` syscall wrapper we should rather use the name of the argument. Maybe `this.fd = args[0].toInt32();` would be clearer?